packages/components/src/button/style.scss

```diff
@@ -101,12 +101,30 @@
 		}
 
 		&:disabled,
-		&[aria-disabled="true"] {
-			color: color(theme(button) tint(30%));
-			background: color(theme(button) shade(30%));
-			border-color: color(theme(button) shade(20%));
+		&:disabled:active:enabled,
+		&[aria-disabled="true"],
+		&[aria-disabled="true"]:active:enabled {
+			color: color(theme(button) tint(40%));
+			background: color(theme(button));
+			border-color: color(theme(button) shade(7%));
 			box-shadow: none;
-			text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.1);
+			text-shadow: none;
+
+			// This specificity is needed to override alternate color schemes in WP-Admin.
```

Comment:
> Hmm actually — I noticed that all the secondary admin color schemes show their box shadows & text shadows. We'll need to override these. 🤔
> 
> ![clicking](https://user-images.githubusercontent.com/1202812/60209043-d0753700-9827-11e9-9095-6ce5bbbd873e.gif)

I pushed a tiny update that fixes this by using a little extra specificity. We can probably count on all buttons in Gutenberg having that `is-button` class, so I _think_ that'll work fine for us. I'd love a gut check from someone else though before merging. @jasmussen do you have a moment to take a look by any chance?

Reply:
I'm not completely sure what to look for. But here's a before:

![before](https://user-images.githubusercontent.com/1204802/60246394-e550d980-98be-11e9-9c56-cbfeb893a551.gif)

Here's an after:

![after](https://user-images.githubusercontent.com/1204802/60246398-e84bca00-98be-11e9-90cf-ed2b22a9c290.gif)

Both look good to me. 

It's always a bummer that we need to add specificity. But until we can improve the code quality of buttons across WordPress, we have to, so on that note it's fine.

Reply:
Ah, sorry — take a look at that when the button is disabled. 🙂

Before: 

![current](https://user-images.githubusercontent.com/1202812/60276063-ef77d580-98c8-11e9-884d-2ec3c49d9d20.gif)

After:

![new](https://user-images.githubusercontent.com/1202812/60276073-f30b5c80-98c8-11e9-8b63-21701b79767a.gif)

This is more in line with the disabled blue button colors in the PR: 

![new2](https://user-images.githubusercontent.com/1202812/60276261-441b5080-98c9-11e9-81c8-3381bbec8c86.gif)

The specificity is probably okay, and if you didn't notice any differences in the non-disabled state, that probably means it didn't break anything important. 😄

Reply:
I just tested the changes as well, @kjellr. Everything worked well. Can I get an approval on this PR?

Reply:
😄 Yep, I didn't want to approve it myself before, since I'd last made some changes to it. If we're both in agreement though, this is good to land! I'll add a comment.